rules.d/34.repo-slackware.yaml

```diff
@@ -0,0 +1,7 @@
+- { name: mozilla-firefox, setname: firefox, family: slackware }
```

Comment:
No need to limit family for this particular rule, this rule is useful if `mozilla-firefox` is met on any repo.